src/Formulate.js

```diff
@@ -73,6 +75,21 @@ class Formulate {
     }
   }
 
+  /**
+   * Produce a deterministically generated id based on the sequence by which it
+   * was requested. This should be *theoretically* the same SSR as client side.
+   * However, SSR and deterministic ids can be very challenging, so this
+   * implementation is open to community review.
+   */
+  nextId (vm) {
+    const path = vm.$route && vm.$route.path && vm.$route.path || false
```

Comment:
```suggestion
    const path = vm.$route && vm.$route.path || false
```

removes duplicate check for `vm.$route.path`